autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -264,6 +285,10 @@ def fit(train_data, label, tuning_data=None, time_limits=None, output_directory=
             Higher levels correspond to more detailed print statements (you can set verbosity = 0 to suppress warnings).
             If using logging, you can alternatively control amount of information printed via `logger.setLevel(L)`,
             where `L` ranges from 0 to 50 (Note: higher values of `L` correspond to fewer print statements, opposite of verbosity levels)
+        epochs : int
```

Comment:
`epochs` should live in scheduler_options as it is an argument for the Hyperband Scheduler. It also should be more precisely named (e.g. `hyperband_epochs`), as epochs could refer to many other things inside autogluon.

Reply:
epochs is not just used in the (HB) scheduler, but also passed to the training function, both in image_classification and in object_detection.

I could remove epochs in those task cases where it is not used outside of the (HB) scheduler, OK?

I just did not want to break external code by changing things.

The argument in HyperbandScheduler is called max_t, not epochs. But for DNN training, this is equal to epochs

Reply:
There is one danger to this. If somebody codes up a new module under task, has an epochs argument, and forgets to copy that to scheduler_options['max_t'], then the scheduler would be inconsistent with the training function.

I'll make the change requested, and add a comment, so hopefully this will not happen.